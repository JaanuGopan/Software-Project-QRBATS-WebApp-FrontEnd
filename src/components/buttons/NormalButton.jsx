import React from 'react';
import './normal-button.css';

const NormalButton = ({ title, icon, handleClick }) => {
  return (
    <div className="row my-1 align-items-center normal-button-container" onClick={handleClick}>
      <div className="col">
        <div className="row py-2">
          <div className="col-auto pe-0">{icon}</div>
          <div className="col-auto fs-6">{title}</div>
        </div>
      </div>
    </div>
  );
};

export default NormalButton;
