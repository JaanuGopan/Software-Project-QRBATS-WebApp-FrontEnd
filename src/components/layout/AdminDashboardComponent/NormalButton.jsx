import React from "react";
import "../../../pages/AdminDashboard/AdminDashboard.css";

const NormalButton = ({ title, titlewithiconicon, handleClick }) => {
  return (
    <div className="NormalButton" onClick={handleClick}>
      {titlewithiconicon}
      <p className="NormalButtonLabel">{title}</p>
    </div>
  );
};

export default NormalButton;
