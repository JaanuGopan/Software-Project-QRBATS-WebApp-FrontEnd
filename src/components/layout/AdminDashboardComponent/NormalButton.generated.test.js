import renderer from 'react-test-renderer';
import React from "react";

import NormalButton from "./NormalButton";

jest.mock("../../../pages/AdminDashboard/AdminDashboard.css");

const renderTree = tree => renderer.create(tree);
describe('<NormalButton>', () => {
  it('should render component', () => {
    expect(renderTree(<NormalButton 
    />).toJSON()).toMatchSnapshot();
  });
  
});