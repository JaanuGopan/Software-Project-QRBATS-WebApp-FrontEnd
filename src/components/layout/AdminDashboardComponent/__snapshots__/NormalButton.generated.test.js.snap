// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NormalButton> should render component 1`] = `
<div
  className="NormalButton"
>
  <p
    className="NormalButtonLabel"
  />
</div>
`;
